cloud/src/recycler/obj_store_accessor.h

```diff
@@ -69,4 +71,44 @@ class ObjStoreAccessor {
     const AccessorType type_;
 };
 
+struct ObjectStoragePathOptions {
+    std::string bucket; // blob container in azure
+    std::string key;    // blob name
+    std::string prefix; // for batch delete and recursive delete
+    std::string_view endpoint;
+};
+
+struct ObjectStorageDeleteExpiredOptions {
+    ObjectStoragePathOptions path_opts;
+    std::function<std::string(const std::string& path)> relative_path_factory;
+};
+
+struct ObjectCompleteMultiParts {};
+
+struct ObjectStorageResponse {
+    ObjectStorageResponse(int r) : ret(r) {}
+    int ret {0};
```

Comment:
Is this for "return code"?
If yes, try to make it `enum class` or pre-define some constants for codes